hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/command/index/TestSecondaryIndex.scala

```diff
@@ -80,40 +83,46 @@ class TestSecondaryIndex extends HoodieSparkSqlTestBase {
           spark.sql(s"insert into $tableName values(3, 'a3', 10, 1002)")
           checkAnswer(s"show indexes from default.$tableName")()
 
-          checkAnswer(s"create index idx_name on $tableName using lucene (name) options(block_size=1024)")()
-          checkAnswer(s"create index idx_price on $tableName using lucene (price options(order='desc')) options(block_size=512)")()
-
-          // Create an index with multiple columns
-          checkException(s"create index idx_id_ts on $tableName using lucene (id, ts)")("Lucene index only support single column")
+          spark.sql(s"create index idx_name on $tableName using secondary_index(name)")
+          checkAnswer(s"show indexes from default.$tableName")(
```

Comment:
since record index is also enabled, shouldn't we show that as well?

Reply:
Addressed